docs/language/src/concepts/translations.md

```diff
@@ -46,8 +58,93 @@ export component MenuItem {
 
 ## Extracting the String from the files
 
-**TODO**
+We provide a `slint-tr-extractor` tool that parse .slint files and can generate a .po file.
+You can run it like so
+
+```sh
+find -name \*.slint | xargs slint-tr-extractor -o MY_PROJECT.pot
+```
+
+This will create a file called `MY_PROJECT.pot`
+(but you should replace MY_PROJECT with your actual project name, see bellow for naming)
```

Comment:
```suggestion
This will create a file called `MY_PROJECT.pot`. Replace MY_PROJECT with your actual project name. To learn how the project name affects the lookup of translations, see the sections below.
```